llvm/test/CodeGen/PowerPC/aix-cc-abi.ll

```diff
@@ -1185,69 +1185,69 @@ define i64 @test_ints_stack(i32 %i1, i32 %i2, i32 %i3, i32 %i4, i32 %i5, i32 %i6
 ; ASM32PWR4-LABEL: test_ints_stack:
```

Comment:
Can you add a signext i8 argument to this test as well?

Reply:
`i1 zeroext` is another case that seems to be missing.

Reply:
> `i1 zeroext` is another case that seems to be missing.

It does seem that this PR departs from other AIX compilers in that it actually clears all but the least significant bit. Other AIX compilers treat this case the same as the `i8 zeroext` case.

Reply:
> It does seem that this PR departs from other AIX compilers in that it actually clears all but the least significant bit. Other AIX compilers treat this case the same as the `i8 zeroext` case.

This has been resolved with the changes; thanks @stefanp-ibm!